spark-bigquery-connector-common/src/test/java/com/google/cloud/spark/bigquery/integration/WriteIntegrationTestBase.java

```diff
@@ -1797,6 +1797,94 @@ public void testOverwriteDynamicPartition_noTimePartitioning() {
         .isEqualTo(Timestamp.valueOf("2023-09-30 12:00:00"));
   }
 
+  @Test
+  public void testOverwriteDynamicPartition_rangePartitioned() {
```

Comment:
According to https://cloud.google.com/bigquery/docs/partitioned-tables#integer_range, values outside the range go into a partition called `__UNPARTITIONED__` and null values go into a partition called `__NULL__`.
Can we do that as well? If it's too complicated, let's call out in the README that the connector doesn't support this behaviour.

Reply:
We are supporting both